lib/runners/processor/golangci_lint.rb

```diff
@@ -0,0 +1,166 @@
+module Runners
+  class Processor::GolangCiLint < Processor
+    include Go
+
+    Schema =
+      StrongJSON.new do
+        let :runner_config,
+            Schema::RunnerConfig.base.update_fields { |fields|
+              fields.merge!(
+                {
+                  target: enum?(string, array(string)),
+                  config: string?,
+                  disable: enum?(string, array(string)),
+                  'disable-all': boolean?,
+                  enable: enum?(string, array(string)),
+                  fast: boolean?,
+                  'no-config': boolean?,
+                  presets: enum?(string, array(string)),
+                  'skip-dirs': enum?(string, array(string)),
+                  'skip-dirs-use-default': boolean?,
+                  'skip-files': enum?(string, array(string)),
+                  tests: boolean?,
+                  'uniq-by-line': boolean?
+                }
+              )
+            }
+
+        let :issue, object(severity: string?)
+      end
+
+    DEFAULT_TARGET = "./...".freeze
+
+    def self.ci_config_section_name
+      # Section name in sideci.yml, Generally it is the name of analyzer tool.
+      "golangci-lint"
+    end
+
+    def analyzer_bin
+      "golangci-lint"
+    end
+
+    def analyzer_name
+      "golangci-lint"
+    end
+
+    def analyze(changes)
+      ensure_runner_config_schema(Schema.runner_config) do |config|
+        @config = config
+        run_analyzer
+      end
+    end
+
+    private
+
+    # [ref] https://github.com/golangci/golangci-lint/blob/master/pkg/exitcodes/exitcodes.go
```

Comment:
[nitpick] Because the commit hash of `master` may change, so how about using a tag (e.g. `v1.23.1`) like this?

```suggestion
    # @see https://github.com/golangci/golangci-lint/blob/v1.23.1/pkg/exitcodes/exitcodes.go
```